docs/api/web-contents.md

```diff
@@ -568,11 +568,14 @@ first available device will be selected. `callback` should be called with
 cancel the request.
 
 ```javascript
-const {app, webContents} = require('electron')
+const {app, webContents, BrowserWindow} = require('electron')
```

Comment:
`webContents` isn't needed in this require anymore

Reply:
I'll take a look later today. I'll test it without the `webContents`.
I'll also change the line
`app.commandLine.appendSwitch('enable-web-bluetooth')`
to
`app.commandLine.appendSwitch('enable-experimental-web-platform-features')`
since chromium is using that flag since the version 56.